client/components/MyScenarios/index.jsx

```diff
@@ -0,0 +1,305 @@
+                            </Table.HeaderCell>
+                            <Table.HeaderCell className="myscenarios__hidden-on-mobile">
+                                Completed
+                            </Table.HeaderCell>
+                        </Table.Row>
+                    </Table.Header>
+                    <Table.Body>
+                        {runsSlice.map(run => {
+                            const {
+                                run_id,
+                                run_created_at,
+                                run_ended_at,
+                                scenario_id,
+                                scenario_title,
+                                cohort_id
+                            } = run;
+                            const completeOrIncomplete = run_ended_at
+                                ? { positive: true }
+                                : { negative: true };
+
+                            const createdAt = moment(run_created_at).fromNow();
+                            const createdAtAlt = moment(
+                                run_created_at
+                            ).calendar();
+
+                            const endedAt = run_ended_at
+                                ? moment(run_ended_at).fromNow()
+                                : '';
+                            const endedAtAlt = run_ended_at
+                                ? moment(run_ended_at).calendar()
+                                : 'This run is not complete';
+
+                            const createdAtDisplay = `${createdAt} (${createdAtAlt})`;
+                            const endedAtDisplay = `${endedAt} (${endedAtAlt})`;
+
+                            const runKey = `${run_id}-${
+                                cohort_id ? cohort_id : scenario_id
+                            }`;
+
+                            const pathname = cohort_id
+                                ? `/cohort/${cohort_id}/run/${scenario_id}/slide/0`
+                                : `/run/${scenario_id}/slide/0`;
+
+                            return (
+                                <Table.Row
+                                    {...completeOrIncomplete}
+                                    key={run_id}
+                                >
+                                    <Table.Cell collapsing>
+                                        <RunMenu
+                                            key={runKey}
+                                            run={run}
+                                            onClick={onRunDataClick}
+                                        />
+                                    </Table.Cell>
+                                    <Table.Cell
+                                        onClick={() => {
+                                            location.href = pathname;
+                                        }}
+                                    >
+                                        <NavLink to={pathname}>
```

Comment:
I think this makes a little more sense as a `Link` rather than a `NavLink` since we don't need the [extra active styles](https://github.com/ReactTraining/react-router/blob/master/packages/react-router-dom/docs/api/NavLink.md) that come with `NavLink`

Reply:
Fixed